spring-mvc-views/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-mvc-views</artifactId>
+    <packaging>war</packaging>
+    <name>spring-mvc-themes</name>
```

Comment:
name, if provided, match artifactId as per guidelines

Reply:
Sure. I missed it during refactoring. Thanks for highlighting that